pkg/api/v1/pod/util.go

```diff
@@ -50,38 +50,65 @@ func FindPort(pod *v1.Pod, svcPort *v1.ServicePort) (int, error) {
 	return 0, fmt.Errorf("no suitable port for manifest: %s", pod.UID)
 }
 
+// ContainerType signifies container type
+type ContainerType int
+
+// DefaultContainers defines default behavior: Iterate containers based on feature gates
+const DefaultContainers ContainerType = 0
+
+const (
+	// Containers is for normal containers
+	Containers ContainerType = 1 << iota
```

Comment:
Maybe `StandardContainers`? Can't think of a great name for this but `Containers` feels too bare.

Reply:
yea, but it aligns with the names in pod spec